coordinator/src/test/java/org/apache/uniffle/coordinator/AccessCandidatesCheckerTest.java

```diff
@@ -38,6 +41,11 @@
 
 public class AccessCandidatesCheckerTest {
 
+  @BeforeAll
```

Comment:
Could we add some tests for Coordinator when it use Kerboros?

Reply:
Yes. I add some tests in previous commits. But I found some conflicts and CI will fail when we use the non-kerberized HDFS and kerberilized HDFS together. Maybe I should dig the cause.